lib/Generate.js

```diff
@@ -396,7 +396,7 @@ function generateMainModule(
     fuzz: isNaN(fuzz) ? 'Nothing' : 'Just ' + fuzz,
     seed: seed,
     report: getReportCode(),
-    paths: testFilePaths.map(sanitizedToString).join(','),
+    paths: testFilePaths.map(sanitizedToString).join(',\n      '),
```

Comment:
```suggestion
    paths: testFilePaths.map(sanitizedToString).join(',\n          '),
```

Looking at the generated file I think 8 spaces is best. (I think the elm compile only complains about too little indentation so adding extra is safe)